app/scripts/metamask-controller.js

```diff
@@ -121,6 +122,16 @@ module.exports = class MetamaskController extends EventEmitter {
         .then((newState) => { cb(null, newState) })
         .catch((reason) => { cb(reason) })
       },
+      importAccountWithStrategy: (strategy, args, cb) => {
+        accountImporter.importAccount(strategy, args)
+        .then((privateKey) => {
+          return keyringController.addNewKeyring('Simple Key Pair', [ privateKey ])
+        })
+        .then(keyring => keyring.getAccounts())
+        .then((accounts) => keyringController.setSelectedAccount(accounts[0]))
```

Comment:
So this selects the first account from the new accounts, not overall? Sounds good, just clarifying.